R/get_info_covidregionaldata.R

```diff
@@ -83,8 +83,16 @@ get_info_covidregionaldata <- function() {
     NA,
     NA,
     "https://raw.githubusercontent.com/pcm-dpc/COVID-19/master/dati-regioni/dpc-covid19-ita-regioni-<date>.csv",
-    "cases_total,deaths_total, tested_total",
-
+    "cases_total, deaths_total, tested_total",
+    
+    "get_regional_data",
+    "france",
+    "r\u00e9gion",
+    "d\u00e9partement",
+    NA,
+    "https://www.data.gouv.fr/fr/datasets/donnees-relatives-aux-resultats-des-tests-virologiques-covid-19/",
+    "cases_new, deaths_new",
```

Comment:
I think `tested_new` is missing here

Reply:
sorry for the late notice

Reply:
Thanks! I intended this to cover only the variables in Level 1 (Regions) anyway, which I think are `cases_new` and `tested_new`. I've updated accordingly in main branch. Let me know if you spot any other errors!